src/pages/experiments/[experimentId]/plots-and-tables/components/PointDesign.jsx

```diff
@@ -9,9 +9,6 @@ import {
 const PointDesign = (props) => {
   const { onUpdate, config } = props;
 
-  const onChange = (e) => {
-    onUpdate({ pointStyle: e.target.value });
-  };
   const onUpdateThrottled = useRef(_.throttle((obj) => onUpdate(obj), 10));
```

Comment:
play arounnd with the seeting here and maybe put it to 50?

Reply:
Set at 50